transform/mattermost-analytics/models/intermediate/web_app/community/int_mm_telemetry_rc__community_tracks.sql

```diff
@@ -8,7 +8,15 @@
 }}
 
     SELECT
-        {{dbt_utils.star(ref('stg_mm_telemetry_rc__tracks'))}},
+        event_id AS event_id,
```

Comment:
Using `except` parameter of [dbt_utils.star](https://github.com/dbt-labs/dbt-utils?tab=readme-ov-file#star-source) should make the code more readable.

Reply:
updated, thanks.